### PR TITLE
[strings] added missing translations for place.*

### PR DESCRIPTION
**Pull request by j13m126**
As usual I used these tools in following order: wikidata, deepl, google translate.
Also looked for pt-BR and removed pt-BR translations identical to pt

### CODE REVIEW

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -14755,6 +14871,8 @@
   [type.place.state.USA]
     en = State
     ar = محافظة
+    be = Cтан
```
Comment (biodranik) [inline data/strings/types_strings.txt:14874]: Штат

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -14755,6 +14871,8 @@
   [type.place.state.USA]
```
Comment (biodranik) [inline data/strings/types_strings.txt:14871]: @vng what is this?

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -14789,15 +14908,18 @@
   [type.place.suburb]
     en = Suburb
     ar = حي سكني
+    be = Прадмесце
```
Comment (biodranik) [inline data/strings/types_strings.txt:14911]: Раён

File: data/strings/types_strings.txt [hunk 4]
```diff
@@ -14823,15 +14945,18 @@
   [type.place.town]
     en = Town
     ar = بلدة
+    be = Гарадок
```
Comment (biodranik) [inline data/strings/types_strings.txt:14948]: Мястэчка

File: data/strings/types_strings.txt [hunk 5]
```diff
@@ -14857,6 +14982,8 @@
   [type.place.village]
     en = Village
     ar = قرية
+    be = Сяло
```
Comment (biodranik) [inline data/strings/types_strings.txt:14985]: Вёска

**Review by biodranik — APPROVED** [review]
@vng PTAL & merge if ok

File: data/strings/types_strings.txt (hunk 2 above)
Comment (vng) [reply data/strings/types_strings.txt:14871]: Видимо раньше как-то особо настраивали отрисовку ..

**Review by vng — APPROVED** [review]

**Pull request merged**